tests/test_frame.py

```diff
@@ -379,12 +379,14 @@ def test_types_drop_duplicates() -> None:
         }
     )
 
-    check(assert_type(df.drop_duplicates({"AAA"}), pd.DataFrame), pd.DataFrame)
     check(assert_type(df.drop_duplicates(["AAA"]), pd.DataFrame), pd.DataFrame)
     check(assert_type(df.drop_duplicates(("AAA",)), pd.DataFrame), pd.DataFrame)
-    check(assert_type(df.drop_duplicates({"AAA": None}), pd.DataFrame), pd.DataFrame)
     check(assert_type(df.drop_duplicates("AAA"), pd.DataFrame), pd.DataFrame)
 
+    if PD_LTE_22:
```

Comment:
```suggestion
    if not PD_LTE_22:
```
You want this test to only be executed if we are in a version after pandas 2.2